src/QMCTools/CMakeLists.txt

```diff
@@ -51,6 +51,10 @@ if(USE_OBJECT_TARGET)
   target_link_libraries(qmcfinitesize qmcparticle qmcutil)
 endif()
 
+if (HAVE_GPAW)
+  configure_file(gpaw4qmcpack ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/gpaw4qmcpack COPYONLY)
```

Comment:
This seems to be an action at install stage.

Reply:
It is. Is there a more suitable way to copy the file over with an intent to install it along with the compiled binaries? The present way does it prematurely, but I'm not an expert on CMake. Also, I think it's debatable whether the python scripts like this should be installed to bin/ or not. I find it convenient to only have one path to add, but for instance PyscfToQmcpack.py doesn't get copied over.

Reply:
I removed the `if`. Making softlink or installing the file doesn't need gpaw, users may install gpaw later.

I changed your `configure_file` to `file(CREATE_LINK)` you can use this an example for other requested changes.

`PyscfToQmcpack.py` doesn't have a main, it is more of a library instead of a standalone tool. We may need to think about how to properly install python files.

Reply:
I need a bit more thinking about the necessity of adding files to `build/bin` leave the problem now. Please make sure the converter test doesn't refer to `gpaw4qmcpack` under `build/bin` but directly to the original file.

Reply:
The converter test now refers to the original file under `src/QMCTools`. It seems that the symbolic link is still copied to `build/bin` before installation stage. Is this an issue? We can also choose to not install the script to `build/bin`, if that's cleaner